proxy-manager/config.yaml

```diff
@@ -29,6 +29,9 @@ map:
   - backup:rw
 backup_exclude:
   - "*/logs"
+options:
+  use_sqlite: false
```

Comment:
I don't want it to be a choice, but a default and only option.
This requires migration.

Reply:
@frenck 
What du you mean with "a default and only option"?
And migration means to have the possibility to switch between mysql and sqlite with data migration into new database?

Reply:
> possibility to switch 

not to switch, just migrate on upgrade to SQLite, and drop MySQL.

Reply:
Understood. Will try to create migration code. Since I haven't use the addon I don't have a mysql database right now. I think that distributing such database is not an option so I need to create one for myself.

Reply:
@frenck How you want to do that migration? Generally there is no easy way how to migrate different DB engines between them.

Easiest will be to keep both options. E.g. if there is env variable for MySQL it will use it, otherwise SQLite.

Reply:
I didn't say it will be easy, it can be done (and we should do it).

Reply:
This may help btw:

https://github.com/Keats/dbmigrate